pkl-gradle/src/main/java/org/pkl/gradle/task/BasePklTask.java

```diff
@@ -166,7 +166,8 @@ protected CliBaseOptions getCliBaseOptions() {
               getTestPort().getOrElse(-1),
               Collections.emptyList(),
               getHttpProxy().getOrNull(),
-              getHttpNoProxy().getOrElse(List.of()));
+              getHttpNoProxy().getOrElse(List.of()),
+              "auto");
```

Comment:
I'm pretty sure this means that Gradle will not output any colors, because this is executed as a child process.

Anyway, we should provide the same options in Gradle (auto, never, always). Take a look at how the other options are configured to see what to do here.

Or, leave a TODO comment here so we can follow up in a future PR.